test/worker_test.exs

```diff
@@ -231,6 +231,94 @@ defmodule WorkerTest do
         false
     end
   end
+
+  describe "parsers" do
+    setup do
+      :meck.expect(Crawly.Utils, :send_after, fn _, _, _ -> :ignore end)
+
+      :meck.expect(Crawly.RequestsStorage, :pop, fn TestSpider ->
+        Crawly.Utils.request_from_url("https://www.example.com")
+      end)
+
+      :meck.expect(
+        HTTPoison,
+        :get,
+        fn _, _, _ ->
+          {
+            :ok,
+            %HTTPoison.Response{
+              status_code: 200,
+              body: "Some page",
+              headers: [],
+              request: %{}
+            }
+          }
+        end
+      )
+
+      on_exit(fn ->
+        :meck.unload()
+      end)
+
+      :ok
+    end
+
+    test "when parsers are declared, a spider's parse_item callback is not called" do
+      :meck.expect(TestSpider, :parse_item, fn _response ->
+        :ignore
+      end)
+
+      :meck.expect(TestSpider, :override_settings, fn ->
+        [parsers: []]
+      end)
+
+      Crawly.Worker.handle_info(:work, %{
+        spider_name: TestSpider,
+        backoff: 10_000
+      })
+
+      refute :meck.called(TestSpider, :parse_item, [:_])
+    end
+
+    test "when parsers are declared, adds new requests and items" do
```

Comment:
The test description is not quite clear. I would rephrase it: spiders are operational with custom parsers